kong.conf.default

```diff
@@ -1374,6 +1374,8 @@
                                  # - `strict`: the router will be rebuilt
                                  #   synchronously, causing incoming requests to
                                  #   be delayed until the rebuild is finished.
+                                 #   (This option is deprecated and will be
+                                 #    removed in future releases)
```

Comment:
Agreed with @dndx : If we are going to leave `eventual` as the only option available, we should deprecate `working_consistency` completely, not just the `strict` value.

Reply:
In that case, I have reworded the changelog entry to make that more explicit. (See suggestion).